default.nix

```diff
@@ -0,0 +1,150 @@
+{ pkgs ? (import <nixpkgs> {})
+, version ? builtins.readFile ./version + ".HEAD"
+, src ? ./.
+}:
+
+let
+  # Build a version of nix that works with current hydra master
+  # (Currently, revision 1a714952732c56b4735f65dea49d406aacc7c595 works)
+  nixSrc = pkgs.fetchgit {
+    url = "https://github.com/NixOS/nix";
+    rev = "1a714952732c56b4735f65dea49d406aacc7c595";
+    sha256 = "05nwfzg33v94hrrqr27vyd3lsg5ix0n9m0yxj1m7wa165ycf9qq6";
+  };
+  nixRelease = import "${nixSrc}/release.nix" {};
+  nix = pkgs.lib.overrideDerivation nixRelease.build.${builtins.currentSystem} (_: {
+    # The checks for this revision are broken, they try to create a /nix/var
+    # directory which results in a permission denied error.
+    doInstallCheck = false;
+
+    # Nix's release.nix sets src to null if in nix shell. This means that
+    # if you try to open a nix-shell on this default.nix, nix will fail to build.
+    # We can fix this by overwriting the src.
+    src = pkgs.lib.overrideDerivation nixRelease.tarball (_: { src = nixSrc; });
+  });
+
+  # Hydra currently requires a fork of aws-sdk-cpp with a few custom patches.
+  aws-sdk-cpp = pkgs.lib.overrideDerivation (pkgs.aws-sdk-cpp.override {
+    apis = ["s3"];
+    customMemoryManagement = false;
+  }) (attrs: {
+    src = pkgs.fetchFromGitHub {
+      owner = "edolstra";
+      repo = "aws-sdk-cpp";
+      rev = "local";
+      sha256 = "1vhgsxkhpai9a7dk38q4r239l6dsz2jvl8hii24c194lsga3g84h";
```

Comment:
Hm, that branch is rather stale (apart from that I'd use the commit rev instead of the branch name, because the latter could change), why exactly does it require a fork?